src/storage/rdb.cc

```diff
@@ -1005,3 +1000,35 @@ Status RDB::rdbSaveBinaryDoubleValue(double val) {
   memrev64ifbe(&val);
   return stream_->Write((const char *)(&val), sizeof(val));
 }
+
+Status RDB::rdbSaveZipListObject(const std::string &elem) {
+  // calc total ziplist size
+  uint prevlen = 0;
+  const size_t ziplist_size = zlHeaderSize + zlEndSize + elem.length() +
+                              ZipList::ZipStorePrevEntryLength(nullptr, prevlen) +
+                              ZipList::ZipStoreEntryEncoding(nullptr, elem.length());
+  auto zl = std::make_unique<unsigned char[]>(ziplist_size);
```

Comment:
would an std::string better here? which also avoid the copying in L:1027

Reply:
thank you very much, done